internal/transactionsubmission/manager.go

```diff
@@ -79,9 +67,9 @@ func (so *SubmitterOptions) validate() error {
 		return fmt.Errorf("monitor service cannot be nil")
 	}
 
-	if so.EventProducer == nil {
-		return fmt.Errorf("event producer cannot be nil")
-	}
+	// if so.EventProducer == nil {
+	// 	return fmt.Errorf("event producer cannot be nil")
+	// }
```

Comment:
This was removed on my last merged PR.

Reply:
I don't think so. It's still in the sdp-multitenant branch:
https://github.com/stellar/stellar-disbursement-platform-backend/blob/b5a3319740eee01472aa5e99bd5424c6ab89d77c/internal/transactionsubmission/manager.go#L82-L84